src/main/scala/com/rbmhtechnology/calliope/EventSource.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2015 - 2017 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.calliope
+
+import akka.NotUsed
+import akka.actor.Cancellable
+import akka.stream._
+import akka.stream.scaladsl.{Flow, Keep, Source, SourceQueueWithComplete}
+import akka.stream.stage._
+
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+/**
+  * An EventSource is an Akka Streams Source which reads events from an [[EventReader]] and emits those events downstream on demand.
+  *
+  * The source may be materialized to a SourceQueue which can be used to signal source updates.
+  * If downstream demand exists and no source updates are emitted through the SourceQueue, periodic periodic polling is used to query for event updates.
+  */
+object EventSource {
+  def apply[Out](eventReader: EventReader[Out], bufferSize: Int, pollingInterval: FiniteDuration): Source[EventRecord[Out], SourceQueueWithComplete[Int]] =
+    EventSourceGraph(EventReadFlow(eventReader, bufferSize), pollingInterval)
+}
```

Comment:
Directly replace with `apply` from `EventSourceGraph` i.e. 

```scala
  def apply[Out](eventReader: EventReader[Out], bufferSize: Int, pollingInterval: FiniteDuration): Source[EventRecord[Out], SourceQueueWithComplete[Int]] = {
    ...
    ticks.mergeMat(commitQueue)(Keep.right)
      .buffer(1, OverflowStrategy.backpressure)
      .via(EventReadFlow(eventReader, bufferSize))
  }
```

and then remove `EventSourceGraph`? (I find the `...Graph` a bit misleading)